public/app/features/dashboard/dashgrid/DashboardEmpty.tsx

```diff
@@ -58,6 +58,32 @@ const DashboardEmpty = ({ dashboard, canCreate }: Props) => {
           </Button>
         </div>
         <div className={cx(styles.centeredContent, styles.others)}>
+          {config.featureToggles.vizAndWidgetSplit && (
```

Comment:
NICE TO HAVE:

This component is getting huge because we don't componentize the different panels/sections. Maybe we can just create sub-components within the same file to split them and them easy to read.

Reply:
yes, I agree. I would do that refactor in a new task (maybe regardless of the decision about implementing or not this new feature)